{"name": "Test đăng nhập với dữ liệu từ Excel", "status": "broken", "statusDetails": {"message": "KeyError: 'username'", "trace": "E   KeyError: 'username'\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "parameters": [{"name": "login_data", "value": "('wrong@gmail.com', 123456, 'fail')"}], "start": 1751536892416, "stop": 1751536892417, "uuid": "af56e201-0df3-4a5c-b5b9-5fb8418a6c6e", "historyId": "4d2c63957c68d56aae51aabe08de0ddb", "testCaseId": "8795e2619fcc855f6c4045aeba8c6c68", "fullName": "tests.test_login#test_login_from_excel", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "SPC"}, {"name": "thread", "value": "11420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}